airflow/executors/base_executor.py

```diff
@@ -49,16 +49,16 @@ def queue_command(self, task_instance, command, priority=1, queue=None):
             self.log.info("Adding to queue: %s", command)
             self.queued_tasks[key] = (command, priority, queue, task_instance)
 
-    def queue_task_instance(
-            self,
-            task_instance,
-            mark_success=False,
-            pickle_id=None,
-            ignore_all_deps=False,
-            ignore_depends_on_past=False,
-            ignore_task_deps=False,
-            ignore_ti_state=False,
-            pool=None):
+    def queue_task_instance(self,
```

Comment:
Let's add an explanation why cfg_path is needed and a JIRA for the appropriate fix as discussed.

Reply:
Added.